db/migrate/20160307161402_add_scheduled_reports.rb

```diff
@@ -0,0 +1,13 @@
+class AddScheduledReports < ActiveRecord::Migration
+  def change
+    create_table :scheduled_reports do |t|
+      t.string :name, null: false
+      t.string :frequency, null: false
```

Comment:
I did consider an enum-style integer definition, or even a `char(1)` (since it takes less storage).

I did want to eventually support full crontab-style frequency values though, for arbitrary schedules, and that would require a string. I.e. "send this to me on MWF" or "every other week" or "every day at noon".